### ISSUE TITLE
Exclude KOReader images from "My Books"

### ISSUE THREAD

**Issue by kahveciderin** [koreader/koreader#5937]
* KOReader version: 2020.03
* Device: Kobo Libra H2O

#### Issue
I installed KOReader and Plato on my kobo libra H2O and everything works perfectly except I do not want to see that unnecessary pictures from "My Books" section. I tried to add this lines to Kobo eReader.conf but that makes the KOReader crash:

```
[FeatureSettings]
ExcludeSyncFolders=\\.(?!kobo|adobe).*?
```

#### Steps to reproduce

After installing KOReader, add those lines to Kobo eReader.conf:

```
[FeatureSettings]
ExcludeSyncFolders=\\.(?!kobo|adobe).*?
```

##### `crash.log` (if applicable)
none

**Comment by Frenzie** [comment]
Sounds the same as https://github.com/koreader/koreader/issues/5918

For some reason Nickel deletes stuff, so you'll have to add it again.

**Comment by NiLuJe** [comment]
Yep, see #5430 

If the filter isn't in place *before* the initial installation, they get indexed, and when you finally put the filter in place, they get de-indexed, which involves deleting the actual files.

TL;DR: Yep, a reinstall will fix it.
